include/deal.II/base/index_set.h

```diff
@@ -218,6 +218,16 @@ class IndexSet
   bool is_contiguous () const;
 
   /**
+   * Return whether the IndexSets are ascending with respect to MPI process
+   * number, i.e., each index is contained in exactly one IndexSet,
+   * the first indices are contained in the IndexSet of the first MPI process,
+   * the second indices are contained in the IndexSet of the second MPI process
+   * and so on.
+   * In case there is only one MPI process, this is always true.
```

Comment:
I haven't read down to the implementation yet, but it should only be true if the index set is complete.